include/aspect/geometry_model/spherical_shell.h

```diff
@@ -95,6 +95,18 @@ namespace aspect
 
           /**
            * @copydoc Manifold::normal_vector()
+           *
+           * We fudge here, but for a good reason. What the function is supposed
+           * to compute is the normal vector to the surface. This *should* be the
+           * normal to the surface with topography, but instead we return the
+           * normal to the undeformed surface -- i.e., the radial direction. This
+           * is, in particular, used to compute no-flux boundary conditions,
+           * for which we want to impose something resembles a boundary
+           * condition that allows for plate-like motion -- that is, we need
```

Comment:
Is this always the right thing to do or just for the problem we are looking at right now? @jdannberg ?

Reply:
I could see cases where one would not want this. For example, if we model flow through a pipe and (ab)use the initial topography plugin to define the upper boundary of the pipe. So, maybe someone does not want to include the lithosphere in their model and only looks at flow through the asthenosphere, and then the initial topography corresponds to the lithosphere depth. 

But that is not a very common case, I would think.